SEC-70 - The definition of TreasuryBill in Bonds is incomplete

ElisaKendall opened the pull request:
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Augmented the definition of TreasuryBill with an explanatory note and added that U.S. Treasury securities are issued in U.S. dollars in a restriction

Fixes: #1723 


## Checklist:

- [x] I'm familiar with the [FIBO developer quide](https://github.com/edmcouncil/fibo/blob/master/CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

rivettp commented on SEC/Debt/Bonds.rdf:
```diff
@@ -1036,6 +1039,7 @@
 		<rdfs:label>treasury bill</rdfs:label>
 		<skos:definition>short-term zero coupon treasury obligation with a maturity ranging from one to twelve months</skos:definition>
 		<fibo-fnd-utl-av:abbreviation>T-bill</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:explanatoryNote>The pricing of T-Bills is unique among government debt issues. Rather than providing interest payments as Treasury Bonds or Notes do, T-Bills are sold at a discount, and the entire return is realized upon maturity. The interest rate earned on T-Bills is equal to the difference between the purchase price and maturity value, divided by the maturity value.</fibo-fn ...
```
a) should say "unique among US government debt issues"
b) should say that the interest rate is annualized (I assume)
c)  if it's tradable what's the price then?

jfgemski replied:
more on c: T-Bills are tradable.  "Some of the other ways to buy Treasuries include ETFs, money market accounts, and the secondary market." -- from Investopedia

tahoeblue commented:
Regarding Pete’s question (‘c’) about price:  

U.S. Treasuries have a par value, and are sold at auction based upon winning bids involving discounts to the par value.  Once “on the run” (in the secondary market) the “price” of U.S. Treasuries is simply what buyers and sellers are willing to negotiate, and this will either be at a premium to, or discount from, the par value (based on current interest rate market expectations).  

> On Feb 9, 2022, at 7:48 AM, Pete Rivett ***@***.***> wrote:
> 
> 
> @rivettp requested changes on this pull request.
> 
> In SEC/Debt/Bonds.rdf <https://github.com/edmcouncil/fibo/pull/1724#discussion_r802808525>:
> 
> > @@ -1036,6 +1039,7 @@
>  		<rdfs:label>treasury bill</rdfs:label>
>  		<skos:definition>short-term zero coupon treasury obligation with a maturity ranging from one to twelve months</skos:definition>
>  		<fibo-fnd-utl-av:abbreviation>T-bill</fibo-fnd-utl-av:abbreviation>
> +		<fibo-fnd-utl-av:explanatoryNote>The pricing of T-Bills is unique among government debt issues. Rather than providing interest payments as Treasury Bonds or Notes do, T-Bills are sold at a discount, and the entire return is realized upon maturity. The interest rate earned on T-Bills is equal to the difference between the purchase price and maturity value, divided by the maturity value.</fibo-fnd-utl-av:explanatoryNote>
> a) should say "unique among US government debt issues"
> b) should say that the interest rate is annualized (I assume)
> c) if it's tradable what's the price then?
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/edmcouncil/fibo/pull/1724#pullrequestreview-877668337>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AB72TFTZIM35H5AZCEPI4UDU2KEEJANCNFSM5NX43JSA>.
> Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
> You are receiving this because your review was requested.
>

ElisaKendall commented:
@rivettp @tahoeblue I think I've addressed this.  I added details from the treasury direct site, as well as links - hoping this addresses concerns.  Please re-review when you have a moment.
